Assets/Scripts/Abilities.cs

```diff
@@ -214,4 +214,328 @@ private IEnumerator _flashlightAbility(PlayerCharacterStats stats)
             EnableButtons();
         }
     }
+
+
+	public void CoveringFire(PlayerCharacterStats stats) { if (stats.Actionsleft > 0) StartCoroutine(_CoveringFire(stats)); }
+	private IEnumerator _CoveringFire(PlayerCharacterStats stats)
```

Comment:
This doesn't appear to do anything.

Reply:
The ability is setting the player to be on covering fire and the effects are in enemy turn script, where the enemies check if they are in range and line of sight of a player who is using covering fire and take damage if they are